hadoop-tools/hadoop-azure/dev-support/testrun-scripts/config_support.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+FILE=$1
+
+if [ ! -f "$FILE" ]; then
+    echo "Error: File '$FILE' not found. Exiting...."
+    exit 1
+fi
+
+if [[ "$1" != *.xml ]]; then
+    echo "The file provided is not an XML file. Exiting...."
+    exit 1
+fi
+
+OUTPUT_FILE="abfs-converted-config.xml"
+cp "$FILE" "$OUTPUT_FILE"
+
+endpoint="dfs"
```

Comment:
should it be .dfs. as we have seen as dfs can be a part of the container name as well